components/templates/Test.tsx

```diff
@@ -75,21 +73,23 @@ export const TestExpanded = function(props: TestProps) {
     else return "black"
   }
 
-  const { data, error } = useSWR(
-    `${process.env.publicDeltaCore}/api/v1/check_if_more_than_five_failed_in_the_last_ten_runs/test_id/${children.test_id}`,
-    fetcher
-  )
+  //TODO: replace this with using test.is_flaky property of test
+  
+  // const { data, error } = useSWR(
+  //   `${process.env.publicDeltaCore}/api/v1/check_if_more_than_five_failed_in_the_last_ten_runs/test_id/${children.test_id}`,
+  //   fetcher
+  // )
```

Comment:
commenting out as I forgot there is a breaking change on the backend

Reply:
Thanks!